docs/cmdline-opts/data-binary.d

```diff
@@ -9,5 +9,10 @@ If you start the data with the letter @, the rest should be a filename.  Data
 is posted in a similar manner as --data does, except that newlines and
 carriage returns are preserved and conversions are never done.
 
+Like --data the default content-type sent to the server is
+application/x-www-form-urlencoded. If you want the data to be treated as
+arbitrary binary data by the server then set the content-type to octet-stream:
+-H "Content-Type: application/octet-stream".
```

Comment:
If you use `--header` literally instead of ` -H` in the body text, it will get highlighted/linked correctly in the output.